challenges/objects-arrays.js

```diff
@@ -55,11 +83,15 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for(let info = 0; info < graduates.length; info++) {
+  contactInfo.push(`${graduates[info].first_name} ${graduates[info].email}`);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+
```

Comment:
If you come back to this, you'll need to use the `.includes()` array method. I know they said not to use any built-in array methods, but this one is OK :smiley: